components/sensors.json

```diff
@@ -110,5 +110,9 @@
   "eco2": {
     "displayName": "Estimated C02",
     "defaultPeriod": 900
+  },
+  "unitless-percent": {
+    "displayName": "Unitless Percentage",
```

Comment:
Maybe should just be "Percentage"? My thinking is that "Unitless" shouldn't ever be seen by end users, but plain old "Percentage" may actually be good enough for some uses in the future. Up to you!

Reply:
Let's keep this, humidiy can also be expressed as a descriptive percentage .